### PR TITLE
First working example

### PR DESCRIPTION
**Pull request by garraldo**

### CODE REVIEW

File: VisionScanner.podspec [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+func startScan(completion:).
+                       DESC
+
+  s.homepage         = 'https://github.com/aldo-dev/VisionScanner.git'
+  s.license          = { :type => 'MIT', :file => 'LICENSE' }
+  s.author           = { 'ALDO Inc' => 'aldodev@aldogroup.com' }
+  s.source           = { :git => 'https://github.com/aldo-dev/VisionScanner.git', :tag => s.version.to_s }
+
+  s.ios.deployment_target = '11.0'
+  s.swift_version   = '4.2'
```
Comment (piyush9290) [inline VisionScanner.podspec]: Can we have this with 4.0? As mFind is still on 4.0.
Or we can update mFind to 4.2.
What you think @AlexHmelevskiAG @garraldo

File: VisionScanner/Classes/VisionScannerViewController.swift [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+        guard captureVideoOutput != nil else {
+            logger?.log(message: "Pause: No captureVideoOutput present.")
+            completion?()
+            return
+        }
+        
+        logger?.log(message: "Scanner paused")
+        currentState = .paused
+        self.requests.removeAll()
+        captureVideoOutput.setSampleBufferDelegate(nil, queue: DispatchQueue.global(qos: DispatchQoS.QoSClass.default))
```
Comment (piyush9290) [inline VisionScanner/Classes/VisionScannerViewController.swift]: `public class func global(qos: DispatchQoS.QoSClass = default) -> DispatchQueue`
You could have used DispatchQueue.global() as as in API it's already default

File: VisionScanner/Classes/VisionScannerViewController.swift [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+            completion?()
+            return
+        }
+        
+        logger?.log(message: "Scanner started")
+        if !captureSession.isRunning {
+            captureSession.startRunning()
+        }
+        
+        captureVideoOutput.setSampleBufferDelegate(self, queue: DispatchQueue.global(qos: DispatchQoS.QoSClass.default))
```
Comment (piyush9290) [inline VisionScanner/Classes/VisionScannerViewController.swift]: Same as above

File: VisionScanner.podspec (hunk 1 above)
Comment (garraldo) [reply VisionScanner.podspec]: I don't think this is needed. The language for the pods is independent of the language of the project where you include it. No? I'm not sure.

File: VisionScanner/Classes/VisionScannerViewController.swift (hunk 2 above)
Comment (garraldo) [reply VisionScanner/Classes/VisionScannerViewController.swift]: Yes, we could;d, but this gives more visibility as to which priority we are assigning.

**Pull request merged**